source-doc/devutils.md

```diff
@@ -10,8 +10,9 @@ Git Utilities
 =============
 
   - `tools/git/hooks/pre-commit`: Copy this to your local repository's `.git/hooks` directory for some automatic
-    housekeeping at each commit. Most notably, this will auto-increment the version number if you commit any changes to
-    the source code. It will also clean up whitespace and do a sanity check of the file header comments.
+    housekeeping at each commit. Most notably, this will auto-increment the version number for offocial pre-releases,
```

Comment:
Thanks - What I get for thinking I can make even a small change without spell checking... :-( I'll fix then re-base to your vm performance commit from yesterday. FYI. I was busy yesterday, and will be busy today and likely for a couple days with life not POV-Ray related.